Makefile

```diff
@@ -58,7 +58,8 @@ bower_components:
 	$(PYC) $^
 
 %.out: %.pyc
-	$(PYTHON) $^ > $@ 2>&1
+	# $(PYTHON) $^ > $@ 2>&1
+	$(PYTHON) $^ &> $@
```

Comment:
On a clean project (after doing ```make clean```), I have to run ```make test``` three times before the makefile stops terminating with an error, and begins execution of tests. I still haven't found a work around to this.